docs/modules/ROOT/pages/how-tos/add-servala-control-cluster.adoc

```diff
@@ -0,0 +1,162 @@
+= Add a servala control cluster
+
+Currently service clusters run as vclusters on one of the CSP's service clusters.
+Usually it runs on the first provisioned cluster.
+Setting up a new control plane will involve configurations on the service cluster as well as on the vcluster itself.
+
+== Setup the vcluster on the service cluster
+
+This configuration has to be added to the service cluster, in order for a vcluster to be provisioned.
+
+The base service cluster configuration also needs to be applied as well.
+
+=== Register cluster
+
+Register the new cluster on the https://control.vshn.net/syn/lieutenantclusters/vshn-lieutenant-prod[control panel].
+
+ID: See https://kb.vshn.ch/app-catalog/adr/0031-naming-scheme-for-servala-cluster-names-and-urls.html#_decision[ADR]
```

Comment:
```suggestion
ID: See xref:adr/0031-naming-scheme-for-servala-cluster-names-and-urls.adoc#_decision[ADR]
```

Use the `xref:` scheme as we're interlinking in the same docs =) and in the other places as well of course